lib/questions.dart

```diff
@@ -1,24 +1,37 @@
-import 'dart:async';
-
-import 'package:path/path.dart';
-import 'package:sqflite/sqflite.dart';
+import 'package:flutter/material.dart';
+import 'package:percent_indicator/percent_indicator.dart';
 
 class Question {
-  final int questionId;
+  final String questionId;
   final String userId;
   final String question;
   final String option1;
   final String option2;
   final String option3;
   final String option4;
-  Question(
-      {this.questionId,
-      this.userId,
-      this.question,
-      this.option1,
-      this.option2,
-      this.option3,
-      this.option4});
+  final String option5;
+  final String vote1;
+  final String vote2;
+  final String vote3;
+  final String vote4;
+  final String vote5;
```

Comment:
better to make it list

Reply:
You are right, I was just being lazy :P Fixed it.